src/calliope/example_models/national_scale/model.yaml

```diff
@@ -4,27 +4,28 @@ import: # Import other files from paths relative to this file, or absolute paths
   - "scenarios.yaml" # Scenario and override group definitions
 
 # Model configuration: all settings that affect the built model
-model:
-  name: National-scale example model
-
-  # What version of Calliope this model is intended for
-  calliope_version: 0.7.0
-
-  # Time series data path - can either be a path relative to this file, or an absolute path
-  timeseries_data_path: "timeseries_data"
-
-  subset_time: ["2005-01-01", "2005-01-05"] # Subset of timesteps
-
-# Run configuration: all settings that affect how the built model is run
-run:
-  solver: cbc
-
-  ensure_feasibility: true # Switches on the "unmet demand" constraint
-
-  bigM: 1e6 # Sets the scale of unmet demand, which cannot be too high, otherwise the optimisation will not converge
-
-  zero_threshold: 1e-10 # Any value coming out of the backend that is smaller than this (due to floating point errors, probably) will be set to zero
-
-  mode: plan # Choices: plan, operate
-
-  objective_options.cost_class: { monetary: 1 }
+config:
+  init:
+    name: National-scale example model
+    # What version of Calliope this model is intended for
+    calliope_version: 0.7.0
+    # Time series data path - can either be a path relative to this file, or an absolute path
+    timeseries_data_path: "timeseries_data"
+    subset_time: ["2005-01-01", "2005-01-05"] # Subset of timesteps
+
+  build:
+    ensure_feasibility: true # Switches on the "unmet demand" constraint
+    mode: plan # Choices: plan, operate
+
+  solve:
+    solver: cbc
+    zero_threshold: 1e-10 # Any value coming out of the backend that is smaller than this (due to floating point errors, probably) will be set to zero
+
+parameters:
+  objective_cost_class:
```

Comment:
Thinking of #484 , this could also default to `data:1, index: [monetary], dims: costs` unless otherwise specified

Reply:
hmm. perhaps.